Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -185,7 +189,6 @@ static bool IsCheckMateOnNextMove(Board board)
         }
         
         Console.WriteLine($"It has to be checkmate for {(board.IsWhiteToMove ? "Black" : "White")} ");
```

Comment:
I guess we should remove this line also as soon as it will be written by the bot both for our predicted move and for opponent predicted move. Thus, there will be a mess and such information will not be helpful.

Reply:
@niklappo fixed :)